scripts/release.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+if [ $# -eq 0 ]
+  then
+    echo "Missing argument with the new version to release"
+    exit 1
+fi
+
+git stash
+sed -i "s/## Unreleased/## Unreleased\n\r## $1/g" CHANGELOG.md
+sed -i "s/\"version\": \".*\",/\"version\": \"$1\",/g" package.json
+yarn
+yarn package
+git add CHANGELOG.md package.json
+git commit -m "Prepare version $1"
+git tag $1
+npmrc npm
+yarn publish ./build/dist/sonar-ui-common-v$1.tgz
```

Comment:
If you want to try locally without really doing a release, comment this

Reply:
**Careful!!** Also comment the `git stash` one 😅 !!

Reply:
Or even better, create a local commit with these lines commented out 😉 .

Reply:
git stash is ok no ? it will unstash at the end...

Reply:
`git stash` will stash the lines you commented out. So it will perform the tasks you commented.

Reply:
ohhh good catch! ^^ it was ok on my machine because didn't have the file in git yet! That was close one!

Reply:
Unfortunately I didn't catch it before running it. But when I saw the output of `yarn publish ...` I frantically `^C`'d 😄